src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You linked a SCSS file in your HTML. Browsers do not understand SCSS. They only understand CSS. You should compile your SCSS to CSS and link the CSS file in your HTML.